rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -38,2339 +38,508 @@
 #include <chrono>
 #include <shared_mutex>
 
+#include "sys_net/network_context.h"
+#include "sys_net/lv2_socket.h"
+#include "sys_net/lv2_socket_native.h"
+#include "sys_net/lv2_socket_raw.h"
+#include "sys_net/lv2_socket_p2p.h"
+#include "sys_net/lv2_socket_p2ps.h"
+#include "sys_net/sys_net_helpers.h"
+
 LOG_CHANNEL(sys_net);
 LOG_CHANNEL(sys_net_dump);
 
-template<>
+template <>
 void fmt_class_string<sys_net_error>::format(std::string& out, u64 arg)
 {
 	format_enum(out, arg, [](auto error)
-	{
-		switch (s32 _error = error)
 		{
-#define SYS_NET_ERROR_CASE(x) case -x: return "-" #x; case x: return #x
-		SYS_NET_ERROR_CASE(SYS_NET_ENOENT);
-		SYS_NET_ERROR_CASE(SYS_NET_EINTR);
-		SYS_NET_ERROR_CASE(SYS_NET_EBADF);
-		SYS_NET_ERROR_CASE(SYS_NET_ENOMEM);
-		SYS_NET_ERROR_CASE(SYS_NET_EACCES);
-		SYS_NET_ERROR_CASE(SYS_NET_EFAULT);
-		SYS_NET_ERROR_CASE(SYS_NET_EBUSY);
-		SYS_NET_ERROR_CASE(SYS_NET_EINVAL);
-		SYS_NET_ERROR_CASE(SYS_NET_EMFILE);
-		SYS_NET_ERROR_CASE(SYS_NET_ENOSPC);
-		SYS_NET_ERROR_CASE(SYS_NET_EPIPE);
-		SYS_NET_ERROR_CASE(SYS_NET_EAGAIN);
-			static_assert(SYS_NET_EWOULDBLOCK == SYS_NET_EAGAIN);
-		SYS_NET_ERROR_CASE(SYS_NET_EINPROGRESS);
-		SYS_NET_ERROR_CASE(SYS_NET_EALREADY);
-		SYS_NET_ERROR_CASE(SYS_NET_EDESTADDRREQ);
-		SYS_NET_ERROR_CASE(SYS_NET_EMSGSIZE);
-		SYS_NET_ERROR_CASE(SYS_NET_EPROTOTYPE);
-		SYS_NET_ERROR_CASE(SYS_NET_ENOPROTOOPT);
-		SYS_NET_ERROR_CASE(SYS_NET_EPROTONOSUPPORT);
-		SYS_NET_ERROR_CASE(SYS_NET_EOPNOTSUPP);
-		SYS_NET_ERROR_CASE(SYS_NET_EPFNOSUPPORT);
-		SYS_NET_ERROR_CASE(SYS_NET_EAFNOSUPPORT);
-		SYS_NET_ERROR_CASE(SYS_NET_EADDRINUSE);
-		SYS_NET_ERROR_CASE(SYS_NET_EADDRNOTAVAIL);
-		SYS_NET_ERROR_CASE(SYS_NET_ENETDOWN);
-		SYS_NET_ERROR_CASE(SYS_NET_ENETUNREACH);
-		SYS_NET_ERROR_CASE(SYS_NET_ECONNABORTED);
-		SYS_NET_ERROR_CASE(SYS_NET_ECONNRESET);
-		SYS_NET_ERROR_CASE(SYS_NET_ENOBUFS);
-		SYS_NET_ERROR_CASE(SYS_NET_EISCONN);
-		SYS_NET_ERROR_CASE(SYS_NET_ENOTCONN);
-		SYS_NET_ERROR_CASE(SYS_NET_ESHUTDOWN);
-		SYS_NET_ERROR_CASE(SYS_NET_ETOOMANYREFS);
-		SYS_NET_ERROR_CASE(SYS_NET_ETIMEDOUT);
-		SYS_NET_ERROR_CASE(SYS_NET_ECONNREFUSED);
-		SYS_NET_ERROR_CASE(SYS_NET_EHOSTDOWN);
-		SYS_NET_ERROR_CASE(SYS_NET_EHOSTUNREACH);
+			switch (s32 _error = error)
+			{
+#define SYS_NET_ERROR_CASE(x) \
+	case -x: return "-" #x;   \
+	case x:                   \
+		return #x
+				SYS_NET_ERROR_CASE(SYS_NET_ENOENT);
+				SYS_NET_ERROR_CASE(SYS_NET_EINTR);
```

Comment:
please remove the crazy indentations and other shenanigans that your autoformat added in this file